ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -370,16 +373,28 @@ inline double getCacheMaxAgeDuration(std::string cacheControlData) {
 
 void RSkComponentImage::requestNetworkImageData(ImageSource source) {
   auto sharedCurlNetworking = CurlNetworking::sharedCurlNetworking();
+  networkRequestMutext.lock();
```

Comment:
networkRequestMutext --> networkRequestMutex is this fine, does t in the end is a typo error or has any meaning